apps/server/src/shared/repo/dashboard/dashboard.repo.integration.spec.ts

```diff
@@ -33,7 +35,7 @@ describe('dashboard repo', () => {
 	it('should persist dashboard with gridElements', async () => {
 		const user = userFactory.build();
 		const course = courseFactory.build({ students: [user] });
-		await em.persistAndFlush([course]);
+		await em.persistAndFlush([course, user]);
```

Comment:
just flushj what the erm currently knows, it will flush all known and updated entities